src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -71,6 +71,10 @@ export class DictionaryConcepts extends Component {
         params: { collectionName, type, typeName },
       },
     } = nextProps;
+    localStorage.setItem('dictionaryId', this.props.match.params.collectionName);
```

Comment:
Why are you storing all these items in the local storage?

Reply:
@dondrzzy  initially there was localStorage.getItem but it was not set. That is why I decided to set the Item fast.

Reply:
> @dondrzzy initially there was localStorage.getItem but it was not set. That is why I decided to set the Item fast.

I believe you meant `first`. That is okay then. Good work